server/schedule/checker/merge_checker.go

```diff
@@ -202,6 +202,11 @@ func (m *MergeChecker) checkTarget(region, adjacent *core.RegionInfo) bool {
 		return false
 	}
 
+	if !checkTargetPeerStoreAvailable(m.cluster, region, adjacent) {
+		checkerCounter.WithLabelValues("merge_checker", "adj-unavailable-peerstore").Inc()
```

Comment:
do we need to judge another neighborhood?

Reply:
> do we need to judge another neighborhood?

I think `MergeChecker.Check` will judge another neighborhood not `checkTarget `